src/modules/base/Module.sol

```diff
@@ -290,4 +290,50 @@ abstract contract Module is
             return false;
         }
     }
+
+    //--------------------------------------------------------------------------
+    // ERC2771 Context Upgradeable
+    // @dev We imitate here the EIP2771 Standard to enable metatransactions
+    // As it currently stands we dont want to feed the forwarder address to each module individually and we decided to move this to the orchestrator
+
+    function isTrustedForwarder(address forwarder)
+        public
+        view
+        virtual
+        returns (bool)
+    {
+        return __Module_orchestrator.isTrustedForwarder(forwarder);
+    }
+
+    function _msgSender()
+        internal
+        view
+        virtual
+        override(ContextUpgradeable)
+        returns (address sender)
+    {
+        if (__Module_orchestrator.isTrustedForwarder(msg.sender)) {
+            // The assembly code is more direct than the Solidity version using `abi.decode`.
+            /// @solidity memory-safe-assembly
+            assembly {
```

Comment:
It is better to use the ERC2771Context methods from OZ instead of writing your own, to avoid these kind of mistakes, toh here as in _msgData. You can override isTrustedForwarder easily